generic/openshift/single-region/procedure/.shellcheckrc

```diff
@@ -0,0 +1 @@
+disable=SC2155
```

Comment:
let's disable it on a global level
- https://github.com/camunda/camunda-deployment-references/blob/main/.pre-commit-config.yaml#L67

Reply:
nevermind, just saw you moved it to folder level.